### ISSUE TITLE
User error!

### ISSUE THREAD

**Issue by JezzaInSingapore** [ValdikSS/openvpn-fix-dns-leak-plugin#3]
Hi, my apologies in advance, but I am having some problems with this on Windows 10 64 bit.

I have put the plugin into the "config" file of OpenVPN, and the line of 
plugin fix-dns-leak-64.dll into the config.

However the plug in fails:
PLUGIN_CALL: POST fix-dns-leak-64.dll/PLUGIN_UP status=1

Any suggestions?  I appreciate this is likely my mistake!

**Comment by ValdikSS** [comment]
Please run it in the console, it should write debug information.

**Comment by ValdikSS** [comment]
So, @JezzaInSingapore, could you provide additional information?

**Comment by JezzaInSingapore** [comment]
Thanks, I have found the problem and it is now hugely obvious... but this might be useful to others.

First of all my VPN provider suggested that I disable IPv6.  This was unnecessary.  What I noticed was even though explicit paths were being given, Windows 10 was not loading the plugin.

It came down to a simple fix.  Right click the OpenVPN GUI icon, select properties, advanced and click on Run As Administrator.

This fixed everything and it works great!  Thank you.
